stix_shifter_modules/nozomi/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,555 @@
+{
+  "ip_src": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "src_ip"
+    },
+    {
+      "key": "ipv6-addr.value",
+      "object": "src_ip"
```

Comment:
this mapping will create both ipv4-addr and ipv6-addr. you need a transformer to check the ip version before creating the object

Reply:
The stix-shifter framework does check internally for the ipv4-addr & ipv6-addr pattern matches. Hence tranformer is not required.